server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -51,6 +51,7 @@
 public class TrustedIntermediaryDashboardView extends BaseHtmlView {
   private final ApplicantLayout layout;
   private final DateConverter dateConverter;
+  public static final String OPTIONAL_INDICATOR = " (optional)";
```

Comment:
**suggestion (non-blocking):** Not blocking since this page isn't internationalized, but we should add this to get translated by putting it in the messages file, since we're going to add this in other places as well. I also wonder if we could check if there are use cases where TIs don't speak English and if so, prioritize adding translations for the rest of the text on this page.